jans-cedarling/bindings/cedarling_js/src/engine/generated-wrapper.ts

```
@@ -0,0 +1,126 @@
+  operation: CedarlingOperation,
+  convert: () => T,
+): T {
+  try {
+    return convert();
+  } finally {
+    try {
+      wrapper.dispose();
+    } catch (error: unknown) {
+      throw createSdkError(errorCode.generatedProtocolError, operation, {
+        rawCause: error,
+      });
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)